client/src/components/QA/Question.jsx

```diff
@@ -50,11 +51,12 @@ class Question extends React.Component {
               <tbody>
                 <tr>
                   <td width='5%' className='character'>Q:</td>
-                  <td width='45%'> {body}</td>
+                  <td width='45%' className='question_body'>{body}</td>
                   <td width='30%' className='helpfulness'>
                     <Helpfulness question={question} />
                   </td>
-                  <td width='20%'> | <a href='#!' id={questionId} onClick={this.addAnswerClicked}> Add answer </a></td>
+                  <td width='20%'> |
+                    <a href='#!' id={questionId} onClick={() => this.addAnswerClicked(questionId)}> Add answer </a></td>
```

Comment:
I've used this method before, but along the lines of my other comments, anonymous arrow functions in a render method will create new function instances every time the component renders. One way around this is to declare a variable in your constructor (i.e., `this.selectedQuestionId`) that you can set instead of declaring a local variable (`const questionId = `). That way it will be visible to `addAnswerClicked` without being passed as an argument.